pkg/logs/internal/launchers/container/launcher.go

```diff
@@ -28,6 +28,7 @@ var containerSourceTypes = map[string]struct{}{
 	"docker":     {},
 	"containerd": {},
 	"podman":     {},
+	"cri-o":      {},
```

Comment:
Not sure where it's used exactly, but shouldn't we log something if we receive something that's not used?